package/plugin-gargoyle-i18n-Polish-PL/files/www/i18n/Polish-PL/qr_code.js

```diff
@@ -0,0 +1,47 @@
+﻿/*
+ * UTF-8 (with BOM) Polish-PL text strings for qr_code.sh html elements
+ */
+
+QrCode.QrCodeViewer="Przeglądarka kodów QR";
+QrCode.QrCode="Kod QR";
+QrCode.SelectQrCode="Wybierz kod QR";
+QrCode.HomeWifi="Domowa sieć bezprzewodowa";
+QrCode.GuestWifi="Bezprzewodowa sieć dla gości";
+QrCode.LocalWebAccess="Lokalny dostęp do sieci";
+QrCode.RemoteWebAccess="Zdalny dostęp do sieci";
+QrCode.IndirectWebAccess="Pośredni dostęp do sieci";
+QrCode.LocalWebcamSnapshot="Migawka lokalnej kamery internetowej";
+QrCode.LocalWebcamStream="Lokalny strumień kamery internetowej";
+QrCode.RemoteWebcamSnapshot="Zdalna migawka kamery internetowej";
+QrCode.RemoteWebcamStream="Zdalny strumień kamery internetowej";
+QrCode.IndirectWebcamSnapshot="Pośrednia migawka kamery internetowej";
+QrCode.IndirectWebcamStream="Pośredni strumień kamery internetowej";
+QrCode.WebcamDisabled="Kamera internetowa jest obecnie wyłączona.";
+QrCode.RemoteWebcamDisabled="Dostęp zdalny jest obecnie wyłączony.";
+QrCode.Ssid="SSID";
+QrCode.Url="URL";
+QrCode.Username="Nazwa Użytkownika";
+QrCode.Password="Hasło";
+QrCode.Width="Szerokość";
+QrCode.PrintDocument="Wydrukuj dokument";
+QrCode.DownloadQrCode="Pobierz kod QR";
+QrCode.QrCodeEditor="Edytor kodów QR";
+QrCode.Application="Aplikacje";
+QrCode.SelectApplication="Wybierz aplikację";
+QrCode.Wifi="Sieć bezprzewodowa";
+QrCode.RouterAccess="Dostęp do routera";
+QrCode.WebAccess="Dostęp do sieci";
+QrCode.Webcam="Kamera internetowa";
+QrCode.WebcamSnapshot="Migawka z kamery internetowej";
+QrCode.WebcamStream="Istrumień z kamery internetowej";
```

Comment:
Typo: "Strumień z kamery internetowej", not "Istrumeń..."

Reply:
Thanks!